App/HejDTO/Sources/HejDTO/Newsletter/Newsletter.swift

```diff
@@ -0,0 +1,7 @@
+
+import Foundation
+
+/// https://docs.hejto.pl/#tag/Newsletter
+public struct Newsletter: Codable, Equatable {
```

Comment:
will there be an instance of this type ever created? if not then, please use an `enum` for that :)